src/TConsole.h

```diff
@@ -192,27 +192,24 @@ class TConsole : public QWidget
     void luaWrapLine(int line);
     QString getCurrentLine();
     void selectCurrentLine();
-
     // Returns the size of the main buffer area (excluding the command line and toolbars).
     QSize getMainWindowSize() const;
-
     ConsoleType getType() const { return mType; }
     virtual void setProfileName(const QString&);
-    // In the next pair of functions the first element in the return is an
```

Comment:
TBH I spotted this comment error in passing and fixed it - I didn't feel up to doing a whole PR just for this.